client/components/place/place-list/place-list.component.js

```diff
@@ -8,10 +8,14 @@ angular.module('placeList', ['filterMapType', 'popularTracks', 'ngAnimate'])
         var ctrl = this;
         var places = [];
         var tracks = [];
-        var placeCounter;
+        var placeCounter = 1;
         var trackCounter;
         var placeTypeLength;
         var trackTypeLength;
+        var key;
+        var map;
+
+        var realPlacesType = [];
```

Comment:
You can rename this variable to:
`activePlacesTypes`
`checkedPlacesTypes`
`currentPlacesTypes`